app/locales/en.js

```diff
@@ -295,7 +295,22 @@ export default {
         selectionCriteriaThree: '3. Merit and usefulness of the data expected to result from the project.',
         deadline: '### Deadline\nProject proposals are accepted on a rolling basis. Applications will be reviewed at the beginning of each month.',
         submissionLink: '[SUBMIT A BIOMEDICAL PROPOSAL](https://goo.gl/forms/uUGdO5CpWDNFE5Uz2)'
-      },
+      }
+    },
+    education: {
+      title: '## Education in the Zooniverse',
+      becomeCitizenScientist: 'As a volunteer on these websites, both you and your students can become citizen scientists and citizen researchers, participating in real science and other research. If you or your students think you have make a mistake, don’t worry; even the researchers make mistakes. These projects are set up to have more than one volunteer analyzing each piece of data, thereby eliminating the vast majority of human error. Mistakes are a part of the process, and can even help us evaluate the difficulty of the data. As long as everyone does their best, they are helping!',
+      resources: '### Resources for educators using Zooniverse',
+      zooTeach: '- [ZooTeach](http://www.zooteach.org/) is a repository of lessons and resources for teachers. At ZooTeach, you will find a variety of resources for education, including: guides to projects for students and teachers, teacher-created presentations designed to introduce students to a particular project, and lessons developed to connect your students to projects and research within the context of things they already know.',
+      educationPages: '- Many Zooniverse projects have their own education pages with additional resources for teachers. Resources may include a video tutorial of how to use the project, other helpful documents and videos about the classification process, and education resources related to the research behind the project.',
+      joinConversationTitle: '### Take part in the conversation',
+      joinConversationBody: 'Keep up with the latest Zooniverse educational efforts on the [Zooniverse Blog](http://blog.zooniverse.org/) or by following [&#64;zooteach](https://twitter.com/ZooTeach) on Twitter. You can also talk with other Zooniverse educators and peers interested in using people-powered research in all sorts of learning environments on the [Zooniverse Education Talk board](http://zooniverse.org/talk/16).',
+      howToUseTitle: '### How are educators are using Zooniverse?',
```

Comment:
it's currently like this, so this isn't because of this PR, but this doesn't make sense to me, am I reading it wrong or is this phrase weird?

Reply:
Yeah, good call. I updated the key to `howEducatorsUseZooniverse`, which hopefully sounds a little less weird.

Reply:
sorry for the confusion - I'm referring specifically to "How are educators are using Zooniverse?"

Reply:
I think the second "are" should be omitted?

Reply:
Ah, nice one - must have looked at that half a dozen times and not noticed it. Good catch!